src.ts/account.ts

```diff
@@ -143,13 +143,13 @@ export class Account {
         return this.signAndSendTransaction(deployContract(this._state.nonce, this.accountId, data));
     }
 
-    async functionCall(contractId: string, methodName: string, args: any): Promise<FinalTransactionResult> {
```

Comment:
do you mind adding a comment explaining what amount does? (might as well document the other ones too, although they are more self-explanatory)

Reply:
what would you write about amount?